Makefile

```diff
@@ -531,12 +531,11 @@ ci-%: checkenv-builddir
 rebuild-testrunner-container-images:
 	@echo "--- building testrunner container image"
 	# temporarily pull repo-global yarn lock file into test/test-remote so
-	# that `yarn install` in there respects that (repo root not part of build
-	# context).
+	# that `yarn install` in there respects that.
 	cp -n yarn.lock test/test-remote
-	docker build --rm --force-rm \
-		--tag opstrace/test-remote:$(CHECKOUT_VERSION_STRING) \
-		-f ./test/test-remote/nodejs-testrunner.Dockerfile test/test-remote/
+	docker build . --rm --force-rm \
+		-t opstrace/test-remote:$(CHECKOUT_VERSION_STRING) \
+		-f ./test/test-remote/nodejs-testrunner.Dockerfile
```

Comment:
Oh. This makes the build context huge (entire opstrace directory, with node_modules etc). Would love to see if that is really needed, I was working pretty hard to make this _not_ needed :).

Reply:
It's mainly so that `opstrace/kubernetes` (and its own dependencies) can be copied into the image per below.

FWIW I just tried a `time make rebuild-testrunner-container-images` on `main` and then again on `nick/test-alerts`, clearing the image build cache each time (via `docker system prune -a`):
- main: 1m36s
- nick/test-alerts: 2m22s

So not _too_ bad...